docs/source/how-to/docker.md

```diff
@@ -0,0 +1,44 @@
+# Run Tiled using Docker
+
+To run tiled using docker we must first obtain a docker image containing a tiled installation.
+To do this run 
+
+```
+docker pull ghcr.io/bluesky/tiled:main
+```
+
+In this example we will use the docker image to serve a directory of files as explained [here](../../tutorials/serving-files.md).
+
+First generate a directory of example files using a utility provided by Tiled.
+
+```
+python -m tiled.examples.generate_files example_files/
+```
+
+The docker container runs tiled via [gunicorn](https://gunicorn.org/) which provides horizontal scaling over workers.
+Because of this we cannot configure tiled using commandline arguments and must specify a config file.
+
+Take the following server configuration below:
+```yaml
+# config.yml
+trees:
+  - path: /
+    tree: tiled.trees.files:Tree.from_directory
+    args:
+      directory: "example_files"
+uvicorn:
+  host: 0.0.0.0
+  port: 8000
+authentication:
+  allow_anonymous_access: true
+```
+and serve it using the docker container
+```
+docker run --rm -p 8000:8000 \
+  --mount type=bind,source="$(pwd)/example_files",target=/deploy/example_files \
+  --mount type=bind,source="$(pwd)/config.yml",target=/deploy/config.yml \
+  --env TILED_CONFIG=/deploy/config.yml tiled
+```
```

Comment:
This example seems that it would launch tiled, and avoid gunicorn.

Reply:
The container is named tiled so this will run the container entrypoint which runs gunicorn. Maybe the container should be named something else to make it more clear?

Reply:
Oh dangit...I misread that (thinking it was exec'ing the tiled not defining the container name). Tiled is the right name.